lib/runner/filematcher.js

```diff
@@ -31,7 +33,15 @@ module.exports = {
      * @returns {boolean} true if specified test matches given tag
      */
     match : function (testFilePath, opts) {
-      var test = require(testFilePath);
+      var test;
+      try {
+          test = require(testFilePath);
+      } catch (e) {
+        // failed to load
+          debug('Failed to load file: ' + testFilePath + ' Error: ' + e);
+          console.error(Logger.colors.red(String.fromCharCode(10006) + ' Failed to load file: ' + testFilePath + ' Error: ' + e + '\n'));
```

Comment:
This is one of our favourite additions over the past few months, we can turn it into an actual PR if it would be accepted..